benchmark_runner/jupyterlab/templates/run_workload/how_to_run_workload.md

```diff
@@ -0,0 +1,31 @@
+# To run a workload with custom parameters through JupyterLab terminal, follow these steps:
+
+1. Open the terminal
+
+```
+cd /notebooks/templates/run_workloads
+```
+
+2. Clone the repository if not already cloned: 
+
+```
+git clone https://github.com/redhat-performance/benchmark-runner
+or 
+update the repository with "git pull" if already cloned.
+```
+
+3. Change the directory to benchmark-runner:
+
+```
+cd benchmark-runner
+```
+
+4. **MUST**: Update the necessary environment variables in benchmark_runner/main/environment_variables.py, e.g., ('WORKLOAD', 'vdbench_vm').
```

Comment:
Modifying a file just to run a command isn't a great idea.  Either set the environment variables in the environment or use command line options rather than modifying this file.